Frameworks/CoreGraphics/D2DWrapper.mm

```diff
@@ -32,4 +32,18 @@ HRESULT _CGGetWICFactory(IWICImagingFactory** factory) {
     static HRESULT sHr = CoCreateInstance(CLSID_WICImagingFactory, nullptr, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(&sWicFactory));
     sWicFactory.CopyTo(factory);
     RETURN_HR(sHr);
+}
+
+HRESULT _CGConvertD2DGeometryToFillMode(ID2D1Geometry* geometry, D2D1_FILL_MODE fillMode, ID2D1Geometry** pNewGeometry) {
```

Comment:
Its a shame that we will always do this conversion even if the geometry is in the right fill mode.  Since we know our defaults, shouldn't we only do the conversion if the mode is different?